tests/Aspire.Hosting.Tests/DistributedApplicationTests.cs

```diff
@@ -563,39 +563,6 @@ await foreach (var resource in s.WatchAsync<Executable>(cancellationToken: token
         }
     }
 
-    [LocalOnlyFact("docker")]
```

Comment:
Tests deleted from this file have been superseded by tests in `ApplicationExecutorTests.cs`